drivers/filesystems/fastfat/rw.c

```diff
@@ -873,8 +873,9 @@ VfatRead(
 
 NTSTATUS
 VfatWrite(
-    PVFAT_IRP_CONTEXT IrpContext)
+    PVFAT_IRP_CONTEXT *pIrpContext)
```

Comment:
(For what it's worth, fastfat may need this change, though fastfat_new does not have it...)